startup.sh

```diff
@@ -11,19 +15,27 @@ __     __        _  __         _   _       _        ____  ___
 EOF
 tput sgr0
 
-# Generate PKI and config if necessary
-if test ! -d data; then
-  command -v cfssl >/dev/null || brew install cfssl
-  ./generate/hub-dev-pki.sh
+# Build a docker image with all of our dependencies for future steps
+if ! command -v docker >/dev/null; then
+  >&2 echo "docker: command not found. verify-local-startup requires docker."
+  exit 1
 fi
 
+docker build -t verify-local-startup .
```

Comment:
Why not use docker-compose for these?

Reply:
This shouldn't be built in docker-compose because it is for building the file dependencies, i.e. it terminating correctly is a prerequisite for starting VLS.

Reply:
The other stuff in docker-compose depends on the output of these bits (e.g. the .env files that ./env.sh generates). We could do it in a separate docker-compose.yml, but it wouldn't provide a lot of additional value.